R/utilities.R

```diff
@@ -1113,7 +1113,7 @@ estimate_purity = function(in_maf,
   }else{
     # If no seg file was provided and assume_diploid paramtere is set to true,
     if(assume_diploid){
-      CN_new <- assign_cn_to_ssm(maf_file = in_maf, assume_diploid = TRUE,coding_only=coding_only,genes=genes)
+      CN_new <- assign_cn_to_ssm(maf_file = in_maf, assume_diploid = TRUE,coding_only=coding_only,genes=genes)$maf
```

Comment:
Is the seq list redundant (i.e not needed for downstream code), or what's the reason for dropping the seq list in this PR?

Reply:
I am keeping the list but just have to specify the maf (at the end), otherwise it can throw an error